gulpfile.js

```diff
@@ -1,26 +1,24 @@
-'use strict';
```

Comment:
When I created this project I was not used to npm scripts yet and was familiar with gulp. If I was creating it now, I would avoid it.